test-requirements.txt

```diff
@@ -14,4 +14,4 @@ pytest>= 4.6.3
 tox>= 3.12.1 
 WebTest>=2.0.33
 flake8>= 3.7.7 
-
+jmespath>=0.9.4
```

Comment:
This shouldn't be in `test-requirements.txt` (it's already in `requirements.txt`)